Automattic/wp-calypso
# Plan name change experiment for Pricing Grid

Related to #

## Proposed Changes

This shows the new Plan names based on the assignment. The changes affect the Pricing grids, but can be visible wherever the plan names from the [plans-list.tsx](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-products/src/plans-list.tsx) are used.

## Testing Instructions



* Follow the steps from D129814-code
* Apply this patch D130166-code
* The pricing grids on /plans and /start/plans should show the new plan names

<img width="480" alt="Screenshot 2023-11-28 at 15 54 06" src="https://github.com/Automattic/wp-calypso/assets/2749938/c76419c1-8607-4a9f-99f9-4fa207b719c2">
<img width="480" alt="Screenshot 2023-11-28 at 15 53 44" src="https://github.com/Automattic/wp-calypso/assets/2749938/f25285ec-3f59-43f2-a4a5-59806196199f">


## Pre-merge Checklist
